tmva/tmva/src/MethodDL.cxx

```diff
@@ -1107,13 +1142,17 @@ void MethodDL::TrainDeepNet()
       std::chrono::time_point<std::chrono::system_clock> tstart, tend;
       tstart = std::chrono::system_clock::now();
 
-      Log() << "Training phase " << trainingPhase << " of " << this->GetTrainingSettings().size() << ":" << Endl;
+      Log() << "Training phase " << trainingPhase << " of " << this->GetTrainingSettings().size() << ":    "
+            << "Learning rate = " << settings.learningRate 
+            << " regularization " << (char) settings.regularization 
+            << " minimum error = " << minTestError
+            << Endl;
       if (!fInteractive) {
          std::string separator(62, '-');
          Log() << separator << Endl;
          Log() << std::setw(10) << "Epoch"
                << " | " << std::setw(12) << "Train Err." << std::setw(12) << "Test Err." 
-               << std::setw(12) << "t(s)/epoch" << std::setw(12)  << "Eval t(s)"
+               << std::setw(12) << "t(s)/epoch" << std::setw(12)  << "t(s)/Loss"
```

Comment:
Is this correct? (Time per loss, what does it mean? Sorry if I'm tired and just don't see it!)

Reply:
It means the time to compute the Loss function for both training and testing data.
For a fixed interval of epochs (defined in testRepetitions) we compute the test error on both 
test and training data set. Maybe we should call t(s)/Test ?